automl/google/cloud/automl_v1beta1/__init__.py

```diff
@@ -33,4 +34,8 @@ class PredictionServiceClient(prediction_service_client.PredictionServiceClient)
     enums = enums
 
 
-__all__ = ("enums", "types", "AutoMlClient", "PredictionServiceClient")
+class TablesClient(tables_client.TablesClient):
+    __doc__ = tables_client.TablesClient.__doc__
+
+
+__all__ = ("enums", "types", "AutoMlClient", "PredictionServiceClient", "TablesClient")
```

Comment:
Our bot will try to remove these changes once this is removed to master. Could you add these replacements to the synth.py for AutoML? [This](https://github.com/googleapis/google-cloud-python/blob/5bc8505f1e885971933cb6093eeff7af0b8be0bd/vision/synth.py#L40-L47) is a similar replacement to add helpers to the vision client.

Reply:
Just to be sure I understand: I should back out all changes to __init__.py, and have synth.py add them instead?

If I do so, the tests will break due to imports not working... should I leave them in until we are ready to merge?

Reply:
Nope! You can keep the changes here, we'd just prefer to also add them to synth.py. We have a bot run all the synth.py scripts nightly and open PRs if the generated version is different from what's currently in master.

Since the generator will produce an __init__.py without the `TablesClient` addition, we need to make this addition permanent via a replacement in `synth.py`.